pennylane/ops/functions/equal.py

```diff
@@ -14,9 +14,10 @@
 """
 This module contains the qml.equal function.
 """
-# pylint: disable=too-many-arguments,too-many-return-statements
+# pylint: disable=too-many-arguments,too-many-return-statements, too-many-branches
```

Comment:
Actually it appears that `too-many-branches` its not needed at all currently, so it can just go 😃 

Should I move the existing ones to where they are triggered?